analyzer-models/src/test/java/ffb/analyzer/models/espn/BaseSerializationTests.java

```diff
@@ -62,6 +67,14 @@ protected final File getResourceFile() {
 
     protected abstract String getResourceFileName();
 
+    protected abstract Class<?> getClassUnderTest();
```

Comment:
What about getClassUnderTesting or getClassBeingTested?

Reply:
Sure. I called it `getClassUnderTesting`